services/QuillLMS/app/views/application/_ortto.html.erb

```diff
@@ -5,10 +5,10 @@
     ap3c.cmd = ap3c.cmd || [];
 
     ap3c.cmd.push(function() {
-        ap3c.init('<%= ENV["ORTTO_KEY"] %>', 'https://capture-api.autopilotapp.com/');
```

Comment:
There are a couple of references in secure_headers that can probably be removed: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/config/initializers/secure_headers.rb#L59